### DISCUSSION TITLE
Slug Field not updating on Update

### DISCUSSION THREAD

**Discussion by schonhoff** [laravel/nova-issues#4754] [Ideas & Feature Requests] [answered]
- Laravel Version: v9.26.1
- Nova Version: 4.13.0
- PHP Version: 8.1.9
- Database Driver & Version: MySQL 8.0.28
- Operating System and Version: Windows 10
- Browser type and version: Google Chrome 104.0.5112.102 
- Reproduction Repository: https://github.com/schonhoff/nova-slug

### Description:
Hello,

if we update a resource with a field (title) and a slug field (title_slug) connected to the field (title), updating the field doesn't update the slug field. I would expect this behavior because it updates on key-up on creating an entry.

### Detailed steps to reproduce the issue on a fresh Nova installation:
1.) Make a model (Blog) with 2 fields (title and title_slug)
2.) Make a Nova resource (Blog) for that model with a Text field (title) and a Slug field (title_slug)
```php
//  public function fields(NovaRequest $request)
[...]
Text::make(__('Title'), 'title')
  ->rules('required', 'max:255', 'string'),

Slug::make(__('Title slug'), 'title_slug')
  ->from('title'),
[...]
```
3.) Switch to the Nova backend and create an entry on the Blog resource
4.) Update the fresh created blog entry and update the title field

expected behavior:
The slug field will change as well

current behavior:
The Slug field will not change.

Another topic with the slug field:
If I may ask a question? Is it possible to add German Umlauts Slug support? ä => ae; ö => oe; ü =>; ß => ss ? Would be very nice! I can open a discussion or another issue if necessary. :-)

**Comment by crynobone** [comment, marked as answer]
The existing behavior is expected and coded as such at the moment. Slugs such as those used in a URL  are considered unique and should require manual editing if needed.

**Reply by schonhoff** [reply]
Thanks for the answer! If it is expected, maybe you can change the docs on the slug field, that it only generates a slug on creating an entity? Maybe it will remove some raising questions in the future :-)

What is with the German Umauts do I need to create another discussion/issue?
